src/apps/content-editor/src/app/views/ItemList/SetActions/SetActions.less

```diff
@@ -8,7 +8,7 @@
   padding: 8px 8px 4px 8px;
 
   .I18N {
-    min-width: 250px;
+    min-width: 200px;
```

Comment:
Why did this shift 50px?

Reply:
The UI was breaking for smaller windows, I18N was taking  a lot of extra space then it needed and also shortening resolved the broken UI.

Reply:
Was it breaking before the component swap?

Reply:
perfect